hamilton/driver.py

```diff
@@ -839,14 +917,27 @@ def visualize_path_between(
             if n.name not in node_modifiers:
                 node_modifiers[n.name] = set()
             node_modifiers[n.name].add(graph.VisualizationNodeModifiers.IS_PATH)
+
+        nodes_to_display = set()
+        for n in nodes_for_path:
+            nodes_to_display.add(n)
+
+            if strict_path_visualization is False:
+                for d in n.dependencies:
+                    nodes_to_display.add(d)
+
         try:
             return self.graph.display(
-                nodes_for_path,
+                nodes_to_display,
                 output_file_path,
                 render_kwargs=render_kwargs,
                 graphviz_kwargs=graphviz_kwargs,
                 node_modifiers=node_modifiers,
-                strictly_display_only_passed_in_nodes=strict_path_visualization,
+                strictly_display_only_passed_in_nodes=True,
```

Comment:
```suggestion
                strictly_display_only_passed_in_nodes=strict_path_visualization,
```
this should be this right?

Reply:
Nope -- the only (relatively easy) way to do this was to calculate the entire set of nodes outside then only visualize those. We should remove this parameter and have the function only display the passed in ones. Otherwise you're choosing the set of included nodes in two places and it is quite confusing.